docs/src/modules/ROOT/attachments/akka-code-init.json

```diff
@@ -26,6 +33,13 @@
       "language": "java",
       "url": "https://github.com/akka-samples/ask-akka-agent"
     },
+    {
+      "title": "Hello world agent (Kotlin)",
+      "description": "This sample uses an agent and LLM to generate greetings in different languages. It illustrates how the agent maintains contextual history in a session memory.",
+      "language": "kotlin",
+      "url": "https://github.com/akka-samples/helloworld-agent-kotlin",
+      "level": "Beginner"
+    },
```

Comment:
Adding the helloworld-agent-kotlin at this position to give precedence to the multi-agent. 

But maybe we should not even have the empty-kotlin that high in the list.